### PR TITLE
Feat ERF-57 - Page routing & Sidebar Wrapper

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds the Sidebar functionality as well as routing and query parameters.

### CODE REVIEW

File: src/lib/utils/queryUtil/index.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+interface PageQueryType {
+  latitude: number | null
+  longitude: number | null
+  zoom: number | null
+  places: number[] | null
+  showShadows: boolean | null
+  showTemperature: boolean | null
+  showWind: boolean | null
+  visibleHour: 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | null
```
Comment (dnsos) [inline src/lib/utils/queryUtil/index.ts:9]: This should be from 10 to 21, shouldn't it?

File: src/lib/utils/queryUtil/index.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+interface PageQueryType {
+  latitude: number | null
+  longitude: number | null
+  zoom: number | null
+  places: number[] | null
```
Comment (dnsos) [inline src/lib/utils/queryUtil/index.ts:5]: What would be encoded in `places`? I'm assuming that we would need to an ID to each one of our POI categories. Not a problem, just good to know to keep this in mind for me.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  mode: 'jit',
```
Comment (dnsos) [inline tailwind.config.js:2]: Uhhh, interesting. Almost got lost for me between all the other stuff. I like 👍

**Review by dnsos — APPROVED** [review]
Wow, I'm impressed by this work! Very elegant, everything tested + Storybook entries and a smooth UI. I've got nothing to add and I'm happy to dive even deeper into this code once it is in `main` - surely plenty of stuff to learn.

Great work @vogelino!

File: src/lib/utils/queryUtil/index.ts (hunk 1 above)
Comment (vogelino) [reply src/lib/utils/queryUtil/index.ts:9]: True! Let's merge that with the rest of the logic later! Ok?

File: src/lib/utils/queryUtil/index.ts (hunk 2 above)
Comment (vogelino) [reply src/lib/utils/queryUtil/index.ts:5]: Exactly, this is still dummy data. We should have IDs here!

File: tailwind.config.js (hunk 3 above)
Comment (vogelino) [reply tailwind.config.js:2]: Haha true I forgot to tell you but it made sense for some utilities. I like it too!

**Pull request merged**